configure.ac

```diff
@@ -231,6 +231,12 @@ LIBMESH_TEST_CXX11_NOEXCEPT
 #-----------------------------------------------------
 LT_INIT([disable-static])
 
+# We need to set the no-undefined flag for linking 
+# with libtool, because Windows requires all shared
+# libraries to include all used symbols. This flag
+# should do no harm on other platforms.
+libmesh_LDFLAGS+="-no-undefined"
```

Comment:
I should've noted, libMesh built fine, but GRINS couldn't link against libMesh with that flag in the LD_FLAGS (which we snag from libmesh-config). Agreed it's odd.